aat/src/aat/resources/features/F-099 SearchParty (GlobalSearch)/S-099.4.td.json

```diff
@@ -0,0 +1,40 @@
+{
+  "_guid_": "S-099.4",
+  "_extends_": "Import_Base_data",
+  "title": "Import Definition file with invalid top level case field name in the 'SearchPartyDoB' column of the SearchCriteria tab",
```

Comment:
Did you mean SearchParty tab instead of SearchCriteria tab?

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/1059/commits/d41404b1f9cf0243dc133d9bf075292529c11e74

Reply:
Fixed in bfe74d7